h2o-parsers/h2o-orc-parser/src/test/java/water/parser/ParseTestORCCSV.java

```diff
@@ -20,61 +23,65 @@
     }
 
     @BeforeClass
-    static public void setup() { TestUtil.stall_till_cloudsize(5); }
+    static public void setup() { TestUtil.stall_till_cloudsize(1); }
 
     @Test
     public void testParseOrcCsvFiles() {
-        int f_index = 0;
-        Frame csv_frame = parse_test_file(csvFiles[f_index], "\\N", 0, null);
-        Frame orc_frame = null;
+        Scope.enter();
+        try {
+            for (int f_index = 0; f_index < csvFiles.length; f_index++) {
 
-        if (forceColumnTypes[f_index]) {
-            byte[] types = csv_frame.types();
-
-            for (int index = 0; index < types.length; index++) {
-                if (types[index] == 0)
-                    types[index] = 3;
-            }
-
-            orc_frame = parse_test_file(orcFiles[f_index], null, 0, types);
-        } else {
-            orc_frame = parse_test_file(orcFiles[f_index], null, 0, null);
-        }
+                Frame csv_frame = parse_test_file(csvFiles[f_index], "\\N", 0, null);
+                Frame orc_frame = parse_test_file(orcFiles[f_index], null, 0, null);
 
+                Scope.track(csv_frame);
+                Scope.track(orc_frame);
 
-        // make sure column types are the same especially the enums
-            byte[] csv_types = csv_frame.types();
-            byte[] orc_types = orc_frame.types();
+                // make sure column types are the same especially the enums
+                byte[] csv_types = csv_frame.types();
+                byte[] orc_types = orc_frame.types();
 
-            for (int index = 0; index < csv_frame.numCols(); index++) {
-                if ((csv_types[index] == 4) && (orc_types[index] == 2)) {
-                    orc_frame.replace(index, orc_frame.vec(index).toCategoricalVec().toNumericVec());
-                    csv_frame.replace(index, csv_frame.vec(index).toNumericVec());
+                for (int index = 0; index < csv_frame.numCols(); index++) {
+                    if ((csv_types[index] == 4) && (orc_types[index] == 2)) {
+                        orc_frame.replace(index, orc_frame.vec(index).toCategoricalVec().toNumericVec());
+                        csv_frame.replace(index, csv_frame.vec(index).toNumericVec());
+                    }
                 }
-            }
 
-        assertTrue(TestUtil.isIdenticalUpToRelTolerance(csv_frame, orc_frame, 1e-5));
+                for (int col_index = 0; col_index < 2; col_index++) {
+                    for (int row_index = 0; row_index < orc_frame.numRows(); row_index++) {
+                        long orc = orc_frame.vec(col_index).at8(row_index);
+                        long csv = csv_frame.vec(col_index).at8(row_index);
+                        long diff = orc - csv;
+                        if (abs(diff) > 0) {
+                            Log.info("**orc csv ts differ at row " + row_index + " col " + col_index + " and the difference is "
+                                    + Long.toString(diff));
```

Comment:
Too much code; just publish both orc and csv.
More, this thing should not go to log, I think; it should be in assert... no?